demos/browser/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "./build",
```

Comment:
This build directory specified by `outDir` needs to be same as the webpack output directory right?

Reply:
I don't think so; webpack's ts-loader will do its thing.

Reply:
So with this, two directories get generated first is `build` generated by compiler and the second is `dist` generated by webpack.

Reply:
That's probably ideal: we don't actually want `tsc`'s output written over Webpack's. Perhaps we want `noEmit` here, just using `tsc` for type checking?

Reply:
I think if webpack uses ts-loader for checking and compiling `ts` files, then we should not emit the output files using `tsconfig`, since webpack generates them.